terraform/datadog/certificates.tf

```diff
@@ -0,0 +1,18 @@
+resource "datadog_monitor" "invalid_tls_cert" {
+  name              = "${format("%s Invalid TLS/SSL Certificate", var.env)}"
+  type              = "metric alert"
+  message           = "${format("{{hostname.name}} certificate {{#is_alert}}is invalid!{{/is_alert}}{{#is_warning}}expires in {{value}} days{{/is_warning}}\n\nSee [Team Manual > Responding to alerts > Invalid Certificates](%s#invalid-certificates) for more info.", var.datadog_documentation_url)}"
```

Comment:
This doesn't use the [notification addresses](https://github.com/alphagov/paas-cf/blob/b913ccf868c6349a387801eb78cdfef52fc096b7/terraform/datadog/variables.tf#L16-L22) to actually notify us. It will show in Datadog as triggered, but we won't be alerted by email/PagerDuty. Example usage can be found in `terraform/datadog/concourse.tf:22`